.github/workflows/sonarqube.yml

```
@@ -12,16 +12,71 @@ jobs:
+          pip install -r backend/requirements.txt
+          pip install pytest pytest-django pytest-cov
+
+      - name: Run migrations
+        working-directory: ./backend
+        run: python manage.py migrate
+
+      - name: Run tests and generate coverage report
+        working-directory: ./backend
+        run: |
+          pytest --cov=api --cov-report=xml:coverage.xml
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)